src/app/action/action.controller.js

```diff
@@ -39,17 +46,17 @@ export class ActionController {
 
   modify($person)
   {
-    console.log('modufiy'+$person);
+    console.log('modify'+$person.name);
```

Comment:
do not use `$` for defining variable names. the general convention is all angular services are prefixed with `$` only